core/federated/RTI/rti_lib.h

```diff
@@ -188,27 +169,7 @@ typedef struct RTI_instance_t {
      * Boolean indicating that authentication is enabled.
      */
     bool authentication_enabled;
-
-    /**
-     * Boolean indicating that tracing is enabled.
-     */
-    bool tracing_enabled;
-} RTI_instance_t;
-
-/**
- * The main mutex lock for the RTI.
- */ 
-extern lf_mutex_t rti_mutex;
-
-/**
- * Condition variable used to signal receipt of all proposed start times.
- */
-extern lf_cond_t received_start_times;
-
-/**
- * Condition variable used to signal that a start time has been sent to a federate.
- */
-extern lf_cond_t sent_start_time;
+} federation_RTI_t;
```

Comment:
This is not introduced by your PR, but I don't see the reason for capitalized "RTI" in types and variable names. This might be a good moment to fix it. In snake case, you would just use lower case, so it would be `federation_rti_t`.

Reply:
I thought it was because RTI is an acronym...
But ChatGPT also advised to do so: "When naming a variable in snake case that includes an acronym, it is common practice to treat the acronym as a regular word and follow the general snake case convention. For example, if the acronym is "API," you would name the variable as "api_variable" or "apiValue." By maintaining consistency with the rest of the snake case style, it helps improve the readability and clarity of the variable name."
:-)